lib/standard_api/controller.rb

```diff
@@ -106,7 +106,8 @@ def update
     end
 
     def destroy
-      resources.find(params[:id]).destroy!
+      ActiveRecord::Base.transaction { resources.find(params[:id].split(',')).each(&:destroy!) }
+
```

Comment:
```suggestion
      resources = resources.find(params[:id].split(','))
      ActiveRecord::Base.transaction { resources.each(&:destroy!) }
```

Reply:
This will only open a transaction after if find them, the find could fail and raise a not found error and not need the transaction at all